src/configurations/destinations/mp/ui-config.json

```diff
@@ -1,448 +1,531 @@
+          "note": "While this is checked, our integration automatically sets all traits on identify calls as super properties and people properties if Mixpanel People is checked as well."
         },
         {
-          "type": "dynamicCustomForm",
-          "value": "superProperties",
-          "customFields": [
-            {
-              "type": "textInput",
-              "label": "Property to send as super Properties",
-              "value": "property",
-              "regex": "^(.{0,100})$",
-              "regexErrorMessage": "Invalid Property Name",
-              "required": false,
-              "placeholder": "e.g: residence"
-            }
-          ]
-        },
-        {
-          "type": "dynamicCustomForm",
-          "value": "setOnceProperties",
-          "footerNote": "Set this for those properties who's values are not supposed to change in profile level. Reference: https://developer.mixpanel.com/reference/profile-set-property-once",
-          "customFields": [
-            {
-              "type": "textInput",
-              "label": "Properties to set only once",
-              "value": "property",
-              "regex": "(^\\{\\{.*\\|\\|(.*)\\}\\}$)|(^env[.].+)|^(.{0,100})$",
-              "regexErrorMessage": "Invalid Property Name",
-              "required": false,
-              "placeholder": "e.g: joiningDate"
-            }
-          ]
-        },
-        {
-          "type": "dynamicCustomForm",
-          "value": "peopleProperties",
-          "customFields": [
-            {
-              "type": "textInput",
-              "label": "Traits to set as People Properties",
-              "value": "property",
-              "regex": "^(.{0,100})$",
-              "regexErrorMessage": "Invalid Property Name",
-              "required": false,
-              "placeholder": "e.g: address"
-            }
-          ]
-        },
-        {
-          "type": "dynamicCustomForm",
-          "value": "eventIncrements",
-          "customFields": [
-            {
-              "type": "textInput",
-              "label": "Events to increment in People",
-              "value": "property",
-              "regex": "^(.{0,100})$",
-              "regexErrorMessage": "Invalid Property Name",
-              "required": false,
-              "placeholder": "e.g: Added-to-cart"
-            }
-          ]
-        },
-        {
-          "type": "dynamicCustomForm",
-          "value": "propIncrements",
-          "customFields": [
-            {
-              "type": "textInput",
-              "label": "Properties to increment in People",
-              "value": "property",
-              "regex": "^(.{0,100})$",
-              "regexErrorMessage": "Invalid Property Name",
-              "required": false,
-              "placeholder": "e.g: Cart-Value"
-            }
-          ]
+          "type": "checkbox",
+          "label": "Ignore \"Do Not Track\"",
+          "configKey": "ignoreDnt",
```

Comment:
ignoreDnt is in defaultConfig, If kept under SDKSettings, will this render ?

Reply:
Yes, 
See Screenshot in PR description